setup.py

```diff
@@ -30,6 +30,7 @@
     ],
     extras_require={
     },
+    tests_require=['pytest'],
```

Comment:
So far we only used the python default unittest package. Any specific reason to use pytest?